zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Identity.java

```diff
@@ -0,0 +1,127 @@
+package com.github.onsdigital.zebedee.api;
+
+import com.github.davidcarboni.restolino.framework.Api;
+import com.github.onsdigital.zebedee.authorisation.AuthorisationService;
+import com.github.onsdigital.zebedee.authorisation.AuthorisationServiceImpl;
+import com.github.onsdigital.zebedee.authorisation.UserIdentity;
+import com.github.onsdigital.zebedee.authorisation.UserIdentityException;
+import com.github.onsdigital.zebedee.json.JSONable;
+import com.github.onsdigital.zebedee.json.response.Error;
+import com.github.onsdigital.zebedee.model.ServiceAccount;
+import com.github.onsdigital.zebedee.reader.util.RequestUtils;
+import com.github.onsdigital.zebedee.service.ServiceStore;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.GET;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import static com.github.onsdigital.zebedee.logging.ZebedeeLogBuilder.logError;
+import static com.github.onsdigital.zebedee.logging.ZebedeeLogBuilder.logInfo;
+import static com.github.onsdigital.zebedee.logging.ZebedeeLogBuilder.logWarn;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.apache.http.HttpStatus.SC_UNAUTHORIZED;
+
+@Api
+public class Identity {
+
+	private AuthorisationService authorisationService;
+    private ServiceStore serviceStore;
+
+    static final String AUTHORIZATION_HEADER = "Authorization";
+
+    @GET
+	public void identifyUser(HttpServletRequest request, HttpServletResponse response) throws IOException,
+			UserIdentityException {
+
+		if (request.getHeader(AUTHORIZATION_HEADER) != null) {
```

Comment:
Replace null check with `StringUtils.isNotEmpty` to cover both null and empty cases